cli/tools/lint.rs

```diff
@@ -308,9 +307,9 @@ pub fn format_diagnostic(
       message_line,
```

Comment:
While you looking into it, could you rewrite this conditional? Lots of duplication and not very readable.

Reply:
Sure, how about this? 99a83ff

Reply:
That works, thanks!